### PR TITLE
Modals for individual job recommendations

### PR DESCRIPTION
**Pull request by rpamidimarri**

### CODE REVIEW

File: app/ui/frontend/src/components/ResumeForm.vue [hunk 1]
```diff
@@ -96,20 +123,53 @@ export default {
...
+      var base_url = "http://34.235.155.212:5601/app/kibana#/dashboard/dd5be340-4078-11e8-878a-e7a4605920c3?embed=true&_g=()&_a=(filters:!(),options:(darkTheme:!f),panels:!((col:1,id:f2ba0fc0-36e9-11e8-bc51-3d1df9db5706,panelIndex:1,row:1,size_x:6,size_y:3,type:visualization),(col:7,id:'9d9c34a0-36e9-11e8-bc51-3d1df9db5706',panelIndex:2,row:4,size_x:6,size_y:3,type:visualization),(col:7,id:'2133c180-36e9-11e8-bc51-3d1df9db5706',panelIndex:3,row:1,size_x:6,size_y:3,type:visualization),(col:1,id:'901a10a0-36e8-11e8-bc51-3d1df9db5706',panelIndex:4,row:4,size_x:6,size_y:3,type:visualization)),query:(query_string:(analyze_wildcard:!t,REPLACE_QUERY_TERM)),timeRestore:!f,title:Jobs,uiState:(),viewMode:view)";
+
+      var url_encoded_job_name = encodeURIComponent(job_name);
+      var query_param = "query:'jobType:%20%22".concat(url_encoded_job_name, "%22%20OR%20searchTerm:%20%22", url_encoded_job_name, "%22'");
+      var url = base_url.replace("REPLACE_QUERY_TERM", query_param);
+
+      var iframe_html = '<iframe src="' + url + '" frameborder="0" style="height:1200px;width:100%;display:block;" height="100%" width="100%"></iframe>';
+
+      this.$refs.jobsModalRef.show();
```
Comment (rpamidimarri) [inline app/ui/frontend/src/components/ResumeForm.vue]: there is potential for refactor & cleanup here. But, in the interest of time, I wanted to get in stuff that works.

File: app/ui/frontend/src/components/ResumeForm.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
         <h1>Paste in your resume!</h1>
         <p>
-          <textarea v-model="resume_text" class="resume_upload_form" style="width: 500px; height:500px;"></textarea>
+          <textarea v-model="resume_text" class="resume_upload_form" style="width: 500px; height:200px;"></textarea>
```
Comment (jameslamb) [inline app/ui/frontend/src/components/ResumeForm.vue]: I support this. Found myself wanting to drag it to be smaller anyway

File: app/ui/frontend/static/app.css [hunk 3]
```diff
@@ -33,6 +33,7 @@ footer {
 }
 
 .footer-text {
+    font-size: 8px;
```
Comment (jameslamb) [inline app/ui/frontend/static/app.css]: nice, nice

File: app/ui/frontend/src/main.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
 import Vue from 'vue'
+import BootstrapVue from 'bootstrap-vue'
```
Comment (jameslamb) [inline app/ui/frontend/src/main.js]: oh well done!

**Pull request merged**